### ISSUE TITLE
Classifier output layer

### ISSUE THREAD

**Issue by emitc2h** [aigamedev/scikit-neuralnetwork#202]
Don't you mean Sigmoid for 2-class classification and Softmax for more than 2 classes in the documentation for classifiers: https://github.com/aigamedev/scikit-neuralnetwork/blob/master/docs/guide_model.rst

I was just reading the TensorFlow tutorial here: https://www.tensorflow.org/versions/r0.8/tutorials/mnist/beginners/index.html

and it looks like Softmax is the right thing for the output layer for a multi-class classifier. Plus, Sigmoid doesn't really seem to work in my own tests.

**Comment by AlbyFer** [comment]
Yep, I had the same doubt too. I'm pretty sure is just an error in the phrasing, because the correct output function for a multiclass problem is indeed the Softmax, which is simply the generalisation of the logistic function.
